src/org.xtuml.bp.core/models/org.xtuml.bp.core/ooaofooa/Subsystem/Model Class/Model Class.xtuml

```diff
@@ -762,10 +762,8 @@ INSERT INTO O_TFR
 	'',
 	"ba5eda7a-def5-0000-0000-000000000005",
 	1,
-	'  select any ref_obj related by self->PE_PE[R8001]->EP_PKG[R8000]->S_SYS[R1405]->EP_PKG[R1405]->PE_PE[R8000]->O_OBJ[R8001]
-      where selected.Obj_ID == param.ref_obj_id;
-
-    
+	'select any ref_obj from instances of O_OBJ where selected.Obj_ID == param.ref_obj_id;
```

Comment:
The select any FIO will generate code that uses getGlobal.  This will go outside of the model.  The getVisibleElementsByName was considered but not used as the operation receives an object id, therefore the caller will have already considered visibility.  Performance should actually be better as it will use the fast lookup (hashmap).  This mostly explained in the note.